crates/wash-lib/src/cli/registry.rs

```diff
@@ -93,6 +93,11 @@ pub struct RegistryPushCommand {
     #[clap(short = 'a', long = "annotation", name = "annotations")]
     pub annotations: Option<Vec<String>>,
 
+    /// World name of the component you are pushing. This is only used if you are pushing a binary
+    /// wit component that may contain multiple worlds.
+    #[clap(long = "world", env = "WASH_REG_WORLD")]
```

Comment:
```suggestion
    #[clap(long = "wit-world", env = "WASH_REG_WIT_WORLD")]
```
What do you think about `wit-world` versus `world`? excessive?

Reply:
I think in the context we're working with here, world should be pretty well known. But I'm not super married to it either